apps/chrome/chrome.py

```diff
@@ -1,4 +1,5 @@
 from talon import ctrl, ui, Module, Context, actions, clip, app
+from talon.mac import applescript
```

Comment:
You probably want this change to go in `apps/chrome/mac.py`, right?

Reply:
Makes sense, will fix. (Can you tell I don't use Chrome regularly?)